install/kubernetes/helm/istio/templates/configmap.yaml

```diff
@@ -171,6 +171,10 @@ data:
     {{- if $.Values.global.useMCP }}
     configSources:
     - address: istio-galley.{{ $.Release.Namespace }}.svc:9901
+      {{- if .Values.pilot.configSource.subscribedResources }}
+      subscribedResources:
+        - {{ .Values.pilot.configSource.subscribedResources }}
```

Comment:
how will this work? We are asking for a list here but then passing a single value

Reply:
+1

Reply:
Right now there is only one option in the API and that is the service registy and if not specified it defaults to MCP (full config). But totally agree this has to change to propagate like a proper list. I think what forced me to do that was the way overrides for helm are defined in test here: https://github.com/istio/istio/blob/master/pkg/test/framework/components/istio/config.go#L125 which is not flexible enought for defining a list.